src/sidebar/components/svg-icon.js

```diff
@@ -74,11 +74,17 @@ function SvgIcon({ name, className = '', inline = false }) {
     markup,
   ]);
 
+  const spanProps = {};
+  if (title) {
+    spanProps.title = title;
+  }
```

Comment:
This is fine. I believe setting the `title` unconditionally should work just the same. Ultimately it just assigns the HTML element's `title` property if a non-`null` value is passed or sets that property to an empty string otherwise ([relevant code](https://github.com/preactjs/preact/blob/c3239535f557d1192901f46a3317cc2b2b2fa865/src/diff/props.js#L117)).

Reply:
Thanks. I know I'm going to be touching this component again shortly, so I will clean up this logic when I do so.